src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -467,6 +473,21 @@ const RealismPage = () => {
                             ))}
                         </SelectGroup>
                     </div>
+
+                    <div className="py-4 flex flex-row justify-between items-center">
+                        <span className="text-lg text-gray-300 mr-1">Home Cockpit Mode</span>
+                        <SelectGroup>
+                            {homeCockpitButtons.map((button) => (
+                                <SelectItem
```

Comment:
This setting is binary in nature -- there is no need to have SelectItems that are only 'Enabled' and 'Disabled'; use the Toggle component here instead.

Reply:
Was following the pattern set out by the realisticTiller directly above this for the sake of consistency. I'll switch this to a toggle component.

Reply:
Toggle component is now used.